rse-cmdline-git.Rmd

```diff
@@ -0,0 +1,2296 @@
+# Git at the command line {#rse-cmdline-git}
+
+## Questions {#rse-cmdline-git-questions}
+
+```{r, child="questions/rse-cmdline-git.md"}
+```
+
+## Introduction {#rse-cmdline-git-intro}
+
+In a previous lesson (TODO: provide link),
+we learned the basics of Git using a graphical interface (GitKraken or R Studio).
+Like many computer programs,
+using Git at the command line is a potentially more powerful option,
+as it allows you to access all the features.
+This chapter describes how the basic "add, commit, push, pull" cycle works at the command line,
+before introducing new concepts including branching and pull requests.
+
+
+## Setting up Git {#rse-cmdline-git-setup}
+
+When we use Git at the command line for the first time,
+we need to configure a few things. 
+
+On a command line, Git commands are written as `git verb options`,
+where `verb` is what we actually want to do
+and `options` is additional optional information which may or may not be required for the `verb`.
+Here is how Dracula sets up his new laptop:
```

Comment:
Is this "Dracula" person used throughout this course? Maybe change to your actual name?